Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/crash.ui/SettingsCrashReportViewModel.kt

```diff
@@ -17,9 +20,15 @@ class SettingsCrashReportViewModel @AssistedInject constructor(
     private val crashReportRepository: BugRepository
 ) : CWAViewModel() {
 
-    val crashReports = crashReportRepository.getAll()
+    val crashReports = crashReportRepository.getAll().asLiveData()
 
-    lateinit var selectedCrashReport: LiveData<BugEvent>
+    private val selectedCrashReportMutable: MutableLiveData<BugEvent> by lazy { MutableLiveData() }
```

Comment:
Why is the empty mutable live data being created lazily? No gain here. `Lazy` would be interesting if we init some value for the `MutableLiveData` here.

Reply:
it is created lazily because it is called in SettingsCrashReportDetailsFragment not in the ReportFragment. So as we don't need the data in any case, lazy has gain.